x-pack/plugins/case/server/client/alerts/update_status.test.ts

```diff
@@ -15,17 +15,13 @@ describe('updateAlertsStatus', () => {
 
     const caseClient = await createCaseClientWithMockSavedObjectsClient({ savedObjectsClient });
     await caseClient.client.updateAlertsStatus({
-      ids: ['alert-id-1'],
-      status: CaseStatuses.closed,
-      indices: new Set<string>(['.siem-signals']),
+      alerts: [{ id: 'alert-id-1', index: '.siem-signals', status: CaseStatuses.closed }],
```

Comment:
So each alert will only have one index? Just curious if index should still be a `Set()`